x-pack/solutions/observability/plugins/apm/public/components/app/onboarding/instructions/otel_agent.tsx

```diff
@@ -236,15 +219,18 @@ export function OpenTelemetryInstructions({
       <EuiText size="xs" color="subdued">
         <FormattedMessage
           id="xpack.apm.onboarding.config_otel.description1"
-          defaultMessage="(1) OpenTelemetry agents and SDKs must support the {otelExporterOtlpEndpoint}, {otelExporterOtlpHeaders}, and {otelResourceAttributes} variables; some unstable components may not yet comply with this requirement."
+          defaultMessage="OpenTelemetry agents and SDKs must support the {otelExporterOtlpEndpoint}, {otelExporterOtlpHeaders}, and {otelResourceAttributes} variables; some unstable components may not yet comply with this requirement."
           values={{
             otelExporterOtlpEndpoint: (
               <EuiLink
                 data-test-subj="apmOpenTelemetryInstructionsOtelExporterOtlpEndpointLink"
                 target="_blank"
                 href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/protocol/exporter.md"
               >
-                OTEL_EXPORTER_OTLP_ENDPOINT
+                {i18n.translate(
+                  'xpack.apm.openTelemetryInstructions.otelexporterotlpendpointLinkLabel',
+                  { defaultMessage: 'OTEL_EXPORTER_OTLP_ENDPOINT' }
+                )}
```

Comment:
I thinks these are keys and should NOT be translated, am I right?

Reply:
Ah true, good catch!